### ISSUE TITLE
victorialogs Occasionally _stream in ""

### ISSUE THREAD

**Issue by neo-hu** [VictoriaMetrics/VictoriaMetrics#6042]
### Describe the bug

`/select/logsql/query?limit=10000&query=_time%3A5m`

```
{"_msg":"\u003c30>Apr  1 00:53:53xxx NetworkManager[6782]: \u003cinfo>  [1711904033.5968] dhcp4 (ens2f0v15): activation: beginning transaction (timeout in 45 seconds)","_stream":"{hostname=\"b2222\",product=\"pppp\",receiver=\"syslog\",n1=\"sss\"}","_time":"2024-03-31T16:53:53.628117054Z"}
{"_msg":"\u003c30>Apr  1 00:53:53 n2 lldpd[5608]: MSAP has changed for port genev_sys_6081, sending a shutdown LLDPDUor address","_stream":"","_time":"2024-03-31T16:53:53.511487672Z"}
{"_msg":"\u003c30>Apr  1 00:49:09 n3 lldpd[5672]: MSAP has changed for port genev_sys_6081, sending a shutdown LLDPDU","_stream":"{hostname=\"n5-54\",product=\"p2\",receiver=\"syslog\",sn=\"s2\"}","_time":"2024-03-31T16:49:09.584346292Z"}
```
Second line _stream “” And it's an occasional phenomenon, Refresh and restore 

### To Reproduce

occasional phenomenon

### Version

v1.99.0-cluster

**Comment by zekker6** [comment]
@neo-hu Could you provide a configuration of log shipper which was used when this occurred?
It would help to create an environment to reproduce the issue.

Also, verison you've specified belongs to VictoriaMetrics, not to VictoriaLogs. Could you check which version of VictoriaLogs are you using?

**Comment by neo-hu** [comment]
> @neo-hu Could you provide a configuration of log shipper which was used when this occurred? It would help to create an environment to reproduce the issue.
> 
> Also, verison you've specified belongs to VictoriaMetrics, not to VictoriaLogs. Could you check which version of VictoriaLogs are you using?

**VictoriaLogs**
tag **v1.99.0-cluster**

/victoria-logs-prod -retentionPeriod=30d -storageDataPath=/data -httpListenAddr=:29428

Plus, the filtering conditions are normal,
Without any conditions, there was an accidental exception, but after refreshing, it returned to normal again

**Comment by neo-hu** [comment]
> > VictoriaLogs
> > tag v1.99.0-cluster
> 
> This release was created by mistake in publish scripts and was unpublished shortly after. Could you try switching to an up-to-date version and see if the issue still persists? You can find all current version here - https://hub.docker.com/r/victoriametrics/victoria-logs/tags The latest release is `v0.5.0-victorialogs` - https://hub.docker.com/layers/victoriametrics/victoria-logs/v0.5.0-victorialogs/images/sha256-1fc71cc156adb84a7ce22c7a5721177e4cc0bc5ff1c3ceb1e11ad380fa729014?context=explore

Which tag/branch corresponds to the git repositorie?

**Comment by zekker6** [comment]
@neo-hu Please, find the tag here - https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v0.5.0-victorialogs

**Comment by zekker6** [comment]
FYI: @neo-hu I've managed to reproduce an issue, seems like `_stream: ""` is returned for a few seconds after stream was created(see [this doc](https://docs.victoriametrics.com/victorialogs/keyconcepts/#stream-fields) for details about streams). Is it possible that stream with log lines that you've shared  was created recently?

**Comment by neo-hu** [comment]
> FYI: @neo-hu I've managed to reproduce an issue, seems like `_stream: ""` is returned for a few seconds after stream was created(see [this doc](https://docs.victoriametrics.com/victorialogs/keyconcepts/#stream-fields) for details about streams). Is it possible that stream with log lines that you've shared was created recently?

Yes, there's no problem with history
Is there a way to solve it

**Comment by valyala** [comment]
FYI, this issue can be related to incorrect registration of new IndexDB entries under high data ingestion load. This could prevent from proper registration of new log streams. The issue has been fixed in the commit 2205de23916f0d4df0b5d22b91adb757306883d5 . This commit will be included in the next VictoriaLogs release.

**Comment by valyala** [comment]
FYI, this issue has been fixed in [VictoriaLogs v0.6.0](https://docs.victoriametrics.com/victorialogs/changelog/).
